functions/Test-DbaMaxDop.ps1

```diff
@@ -196,7 +198,8 @@ function Test-DbaMaxDop {
 
 					$dbmaxdop = $database.MaxDop
 
-					$collection += [pscustomobject]@{
+					#$collection +=
```

Comment:
Thanks for the updates and testing claudio! For future reference, there is no need to keep any portion of the collection.

Reply:
🤦‍♂️  Will do another PR in a couple of days to clean this.
Thanks!